packages/TermsAndConditions/TermsAndConditions.jsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState, useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import HairlineDivider from '@tds/core-hairline-divider'
+import DimpleDivider from '@tds/core-dimple-divider'
+import FlexGrid from '@tds/core-flex-grid'
+import Heading from '@tds/core-heading'
+import Box from '@tds/core-box'
+import { colorWhite, colorTelusPurple } from '@tds/core-colours'
+import { FadeAndReveal, Translate } from '@tds/shared-animation'
+
+import safeRest from '../../shared/utils/safeRest'
+import Clickable from '../../shared/components/Clickable/Clickable'
+
+import List from './List/List'
+
+const defaultCopy = {
+  en: {
+    headingClosed: 'View terms and conditions',
+    headingOpened: 'Hide terms and conditions',
+  },
+  fr: {
+    headingClosed: 'Voir les modalités et conditions',
+    headingOpened: 'Masquer les modalités et conditions',
+  },
+}
+
+const getCopy = copy => (typeof copy === 'string' ? defaultCopy[copy] : copy)
```

Comment:
So this would be some sort of toolbox for general component helper methods? I wonder if it could be split off more, just to avoid a million version bumps whenever that toolbox gets updated. Don't feel particularly strongly about that though.